deluge/ui/gtkui/common.py

```diff
@@ -75,7 +75,7 @@ def build_menu_radio_list(value_list, callback, pref_value=None, suffix=None, sh
         item_text = str(value)
         if suffix:
             item_text += " " + suffix
-        menuitem = gtk.RadioMenuItem(group, item_text)
+        menuitem = Gtk.RadioMenuItem(item_text, group)
```

Comment:
menuitem = Gtk.RadioMenuItem.new_with_label(group, item_text) =
TypeError: Must be sequence, not RadioMenuItem

so maybe but needs aditional changes?